index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+
+
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <div class="toolbar-menu">
+            <nav class="menu-items">
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+            <div class="hamburger">
+                <i class="fas fa-bars"></i>
+            </div>
+        </div>
+    </header>
+    <section>
+<div class="header">
+    <div class="header-content">
+        <h1>Hey there. <br>I am Alex.</h1>
+        <h1>I'm a software developer</h1>
```

Comment:
- [ ] kindly give  classes to the h1 elements and style them properly. 
they are styled as such

font-family: Inter;
font-size: 40px;
font-weight: 700;
line-height: 52px;
letter-spacing: 0em;
text-align: center;